simulator/sim_modules/passport/sram4.py

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: 2021 Foundation Devices, Inc. <hello@foundationdevices.com>
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-MAX_QR_VERSION = 24
+MAX_QR_VERSION = 13
```

Comment:
Why was this changed, and why was it defined here and in modpassport-sram4.h?

Reply:
This seems to be defined multiple places, is it possible to consolidate to 1 global constant?

Reply:
I changed it to match the code in `modpassport-sram4.h`, we can't consolidate it yet because `sram4.py` simulates that module from `modpassport-sram4.h` so it's needed. What I really like to do in the future is have a `modpassport-sram4.h` defined for both unix and stm32 ports and using the preprocessor to choose between the native implementation and the hardware one.

Reply:
It's also defined in modules/views/qrcode.py, could that be consolidated?